README.md

```diff
@@ -42,6 +42,18 @@ If you want to use [PROJ](https://proj.org/) in your code, you can if you
 activate the `proj` feature (`cargo build --features=proj`). Then don't forget
 to protect your code with `#[cfg(feature="proj")]`.
 
+### Feature `xmllint`
+`transit_model` is capable of exporting NeTEx France format. In the tests, we're
+automatically verifying that the produced files are matching the NeTEx
+specification.  For that, we're using the tool `xmllint` which can be install
+on Debian with the package `libxml2-utils`. Therefore, these tests are run only
+if you activate them.
+
+```bash
+apt install libxml2-utils
```

Comment:
Perhaps the following commands should be added for this to work.
```
git submodule init
git submodule update
```

Reply:
Thanks. I added some more documentation for that. See d2e3ee2f0d705988720d3f76f60f9f8ad4bbb45a.